### ISSUE TITLE
Sala crashing when using live interface,

### ISSUE THREAD

**Issue by Chrrxs** [Gskartwii/roblox-dissector#67]
[Video of issue](https://dl.dropboxusercontent.com/s/4gv2keec6vpu4c0/MzOnzyjym2.mp4)
When pressing play on a game and beginning to scan Sala crashes.

**Comment by Arisstath** [comment]
roblox has introduced a new update that implements full protocol encryption. sala is on hold at the moment

**Comment by FardMan69420** [comment]
Will this be fixed? If not do you have any recommendations on how I can decrypt these requests in wireshark

**Comment by PerennialWheat** [comment]
> Will this be fixed? If not do you have any recommendations on how I can decrypt these requests in wireshark

I don't think this will be fixed, but if you want to make your own fix others have said: 

1. Hook robloxs network function/method before encryption to dump the client packets (You must reverse engineer the client yourself).

2. patch client cert pinning and perform a mitm on the join game endpoint. (replace server pubkey with your own)

**Comment by FardMan69420** [comment]
Ok that seems like a task that is outside of my skillset
